app/init.php

```diff
@@ -1776,6 +1791,27 @@ function getDevice(string $root, string $connection = ''): Device
     return [];
 });
 
+App::setResource('hasDevelopmentKey', function ($request, $project, $dbForConsole) {
```

Comment:
Just pass the resource, we anyway do this check only in one place. If a must, we can also consider adding a static check to the auth class.